docs/apache-airflow/index.rst

```diff
@@ -104,6 +104,7 @@ unit of work and continuity.
     production-deployment
     backport-providers
     faq
+    Issue reporting and resolution process <issue-triage-process>
```

Comment:
Is this documentation intended for the end user or for a contributor? I think it would be better if this link got added to CONTRIBUTING.rst.

Reply:
I believe it is useful for both end users who report issues and want to understand the process of "what next", as well as for contributors. 
I do think adding this link to the CONTRIBUTING.rst also makes sense, once this is reviewed and published.

Reply:
how often do these rules change?  as often as a new release comes out, or more often? Is it related to specific Airflow version?

Reply:
I am sorry, I don't understand the question.

Reply:
The apache-airflow package is versioned according to the apache-airflow release cycle. If you think it is worthwhile for users to have access to this document, we need to consider whether users will have access to a good version of this document. It seems to me that this page is not related to any particular version of apache-airflow and will therefore be out of date very quickly. It may happen that after the first day of release of a new version of apache-airflow, we will want to update the process a bit, e.g. to add new labels, but this will be a bit difficult because we try to keep the documentation immutable. We only update the documentation when a new version is released.

Please note where this page is posted on the website. If you want to go to this article, you will have to go to airflow.apache.org, click "Documentation", click "Apache Airflow", and. finally click "Issue reporting and resolution process". If you want to post it on airflow.apache.org then it's a better idea to add it to the section ["Community"](http://airflow.apache.org/community/), which already describes the rules for reporting bugs.

Reply:
I agree with @mik-laj. This document is likely to change between releases, so it falls into the same category as CONTRIBUTING.rst or  README. I agree that it should be in GitHub rather than in the generated documentation. The best we can do is to have it in GitHub and link to it from the documentation if we want to make users aware of it's existence. Everything it talks about is GitHub issues, so there is no problem with it being in GitHub and not under `airflow.apache.org`.

Reply:
Got it. 
I removed the change to the documentation index and moved this new .rst file to the top level directory.
I added a reference to it in CONTRIBUTING.rst